packages/ui/.storybook/assets/fonts.scss

```diff
@@ -9,7 +9,4 @@ body {
   font-size: 1rem;
   font-weight: 300;
   line-height: 1.5;
-  color: #34495e;
-  text-align: left;
-  background-color: #eeeeee;
```

Comment:
Maybe as a separate pr?

We'll get a lot of [changes](https://www.chromatic.com/build?appId=64d608ab34b67e621ab50cfd&number=432)